upi/metal/main.tf

```diff
@@ -8,8 +8,6 @@ locals {
 
     # "rd.break=initqueue"
     "coreos.inst=yes",
-
-    "coreos.inst.image_url=${var.pxe_os_image_url}",
     "coreos.inst.install_dev=sda",
```

Comment:
This will still work, but we generally spell it `coreos.inst.install_dev=/dev/sda` now.